### PR TITLE
snapd api: wait longer

### PR DESCRIPTION
**Pull request by dbungert**
While these changes are not supposed to take nearly this long, per LP: #[2034715](https://bugs.launchpad.net/snapd/+bug/2034715) we know that they are, and that some systems will correctly perform the finish_install() step if just given more time.

### CODE REVIEW

**Review by Chris-Peterson444 — APPROVED** [review]
This looks straightforward to me. We may want to reconsider reverting this once the issue in snapd is resolved.

File: subiquitycore/snapd.py [hunk 1]
```diff
@@ -41,13 +41,15 @@ def get(self, path, **args):
         if args:
             path += "?" + urlencode(args)
         with requests_unixsocket.Session() as session:
-            return session.get(self.url_base + path, timeout=60)
+            return session.get(self.url_base + path, timeout=600)
```
Comment (ogayot) [inline subiquitycore/snapd.py:44]: Please consider adding a comment linking to the LP bug.

**Review by ogayot — APPROVED** [review]
LGTM

File: subiquitycore/snapd.py (hunk 1 above)
Comment (ogayot) [reply subiquitycore/snapd.py:44]: I like that it's present in the commit message though, so feel free to disregard if you think it's enoug.

Comment (mwhudson) [reply subiquitycore/snapd.py:44]: Yeah I'd like a comment near the code too please

**Review by mwhudson — APPROVED** [review]

**Pull request merged**